paasta_tools/cli/cmds/logs.py

```diff
@@ -100,9 +103,37 @@ def add_subparser(subparsers):
         default=DEFAULT_SOA_DIR,
         help="define a different soa config directory",
     )
+
+    status_parser.add_argument(
+        '-a', '--from', '--after', dest='time_from',
+        help='The ISO-8601 timestamp to get the logs from. Alternatively, for convenience, time differences can be '
```

Comment:
can you be more specific about what is allowed? 
> The time to get logs from. This can be expressed as an ISO-8601 datestring or any human-readable duration parseable by pytimeparse, such as 5m'